OutOfSchool/OutOfSchool.WebApi.Tests/Models/ParentDraftCreationTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using NUnit.Framework;
+using OutOfSchool.Common;
+using OutOfSchool.Services.Enums;
+using OutOfSchool.Services.Models;
+
+namespace OutOfSchool.WebApi.Tests.Models;
+
+[TestFixture]
+public class ParentDraftCreationTests
+{
+    [Test]
+    public void CreateDraft_ReturnsMaleGender()
+    {
+        // Assert
+        Assert.AreEqual(Gender.Male, Parent.CreateDraft("test").Gender);
+    }
+
+    [Test]
+    public void CreateDraft_ReturnsMaleValidAge()
+    {
+        // Arrange
+        var expectedDate = DateTime.Now.AddYears(-Constants.AdultAge).Date;
+
+        // Assert
+        Assert.AreEqual(expectedDate, Parent.CreateDraft("test").DateOfBirth.GetValueOrDefault().Date);
+    }
```

Comment:
(Flaky test) In this test you check the `Now` date. It will be broken from time to time.

There are 2 issues: 
- if test is running at 23:59:59.xxx the date of `Now` set into `expectedDate` and via `CreateDraft()` could be different day;
- test uses `Now` while implementation is using `UtcNow`, it would work correctly only for UTC timezone, for other timezones you could have problems during several hours of each day.

Solutions:
 - either use `TimeProvider` and `FakeTimeProvider` (added in .Net 8, see [example](https://grantwinney.com/how-to-use-timeprovider-and-faketimeprovider/)) - lot of code changes, good to have everywhere in code, not sure if this PR is a right place
 - or just pass the now datetime to the `CreateDraft()`.

Reply:
> (Flaky test) In this test you check the `Now` date. It will be broken from time to time.
> 
> There are 2 issues:
> 
> * if test is running at 23:59:59.xxx the date of `Now` set into `expectedDate` and via `CreateDraft()` could be different day;
> * test uses `Now` while implementation is using `UtcNow`, it would work correctly only for UTC timezone, for other timezones you could have problems during several hours of each day.
> 
> Solutions:
> 
> * either use `TimeProvider` and `FakeTimeProvider` (added in .Net 8, see [example](https://grantwinney.com/how-to-use-timeprovider-and-faketimeprovider/)) - lot of code changes, good to have everywhere in code, not sure if this PR is a right place
> * or just pass the now datetime to the `CreateDraft()`.

Fixed, now `CreateDraft` accepts `currentTime` parameter as starting point for subtracting years.